src/toolkits/style_transfer/style_transfer.cpp

```diff
@@ -616,23 +615,27 @@ void style_transfer::train(gl_sarray style, gl_sarray content,
 
 std::shared_ptr<MLModelWrapper> style_transfer::export_to_coreml(
     std::string filename, std::map<std::string, flexible_type> opts) {
-  flex_int image_width = read_state<flex_int>("image_width");
-  flex_int image_height = read_state<flex_int>("image_height");
+  const flex_int image_width =
+      read_opts<flex_int>(opts, "image_width", /* delete item */ true);
+  const flex_int image_height =
+      read_opts<flex_int>(opts, "image_height", /* delete item */ true);
+  const flex_int include_flexible_shape =
+      read_opts<flex_int>(opts, "include_flexible_shape", /* delete item */ true);
 
   flex_dict user_defined_metadata = {
       {"model", read_state<flex_string>("model")},
-      {"max_iterations", read_state<flex_int>("max_iterations")},
-      {"training_iterations", read_state<flex_int>("training_iterations")},
+      {"max_iterations", std::to_string(read_state<flex_int>("max_iterations"))},
```

Comment:
Why are we storing these as strings?

Reply:
CoreML expects this metadata as string data so if we don't cast this user_defined_metadata, `export_core_ml` throws an exception.

Reply:
What exception? The implementation is intended to handle this, and to convert to string automatically: https://github.com/apple/turicreate/blob/4a8bbf96c6dfd1b302fc264e58ddc2c04f398431/src/toolkits/coreml_export/coreml_export_utils.cpp#L40

Reply:
Might have conflated two different errors. I reverted and everything worked!